README.md

```diff
@@ -91,7 +91,7 @@ $ cd rmm
 - Create the conda development environment `rmm_dev`
 ```bash
 # create the conda environment (assuming in base `rmm` directory)
-$ conda env create --name rmm_dev --file conda/environments/rmm_dev_cuda11.0.yml
+$ conda env create --name rmm_dev --file conda/environments/full_cuda-11.5_arch-$(arch).yaml
```

Comment:
```suggestion
$ conda env create --name rmm_dev --file conda/environments/rmm_dev_cuda11.5.yaml
```

Reply:
@bdice, the dependency file generator generates the dependency file names based on the CUDA versions and architectures. that's why I changed the file name here. Is there a reason you suggested changing it back?

Reply:
Okay, just trying to align the README file path with the generated/committed filename. They weren’t aligned. Looks like it is `conda/environments/all_cuda-11.5_arch-x86_64.yaml` now.